sncosmo/builtins.py

```diff
@@ -90,7 +90,7 @@ def load_bandpass_remote_um(relpath, name=None):
 
 def load_bandpass_remote_wfc3(relpath, name=None):
     abspath = DATADIR.abspath(relpath)
-    _, wave, trans = np.loadtxt(abspath, unpack=True)
+    wave, trans = np.loadtxt(abspath, unpack=True)
```

Comment:
Hmmm... did the format of the wfc3 files change? If so, the current released versions of sncosmo probably won't be able to load the new files now hosted at sncosmo.github.io.

Reply:
Ah yes I forgot about that, the old files had an index column. Sorry about that I'll issue a new request, it'll have to be to both places unfortunately.

Reply:
Already did the sncosmo.github.io one.

Reply:
Oh you added back in the column? I just fixed load_bandpass_remote_wfc3 in this request, so if you fixed the files back in io then we should be good.

Reply:
Other than it looks like you replaced the files, you didn't add a column right? So the few filters I added into wfc3-uvis aren't there anymore but are in builtins.py

Reply:
I think the easiest thing to do here is just cancel this pull request. The new filters won't go in but that's okay, no other changes to builtins.py and sncosmo.github.io looks right.

Reply:
Sorry for the confusion here.

I reverted the addition of all the files on sncosmo and instead added the same files that used to be hosted on the stsci website (I had them cached locally). These files have the three column format.

Going forward, to add the new filters and new versions of the old filters, we might want to do it in a separate sub-directory of sncosmo.github.io/data, so that old versions of sncosmo continue working. Do you think it is worth updating all the files? Or just adding the new ones? It's not clear to me what the difference is between the old and new versions, besides just formatting.